src/pages/Pages/specials/Index/index.tsx

```diff
@@ -1,13 +1,16 @@
 import ErrorCard from 'pages/Pages/specials/Index/ErrorCard';
-import Changelog from 'pages/Pages/specials/Index/Changelog';
+import ChangelogCard from 'pages/Pages/specials/Index/ChangelogCard';
 import MembersCard from 'pages/Pages/specials/Index/MembersCard';
 import { Groups } from 'types/Enums';
 
+const frontendURL = 'https://raw.githubusercontent.com/tihlde/Kvark/master/CHANGELOG.md';
+const backendURL = 'https://raw.githubusercontent.com/tihlde/Lepton/master/CHANGELOG.md';
```

Comment:
Lurt å lage konstanter av disse url'ene! Det er alltid lurt å ha variabel-navnene til konstanter i store bokstaver slik at man andre steder i koden kan se at det er en konstant.
```suggestion
const BACKEND_URL = 'https://raw.githubusercontent.com/tihlde/Lepton/master/CHANGELOG.md';
```